test/e2e/storage/pd.go

```diff
@@ -428,6 +428,8 @@ var _ = framework.KubeDescribe("Pod Disks", func() {
 			By("Cleaning up PD-RW test env")
 			podClient.Delete(host0Pod.Name, metav1.NewDeleteOptions(0))
 			detachAndDeletePDs(diskName, []types.NodeName{host0Name})
+			framework.WaitForNodeToBeReady(f.ClientSet, string(host0Name), nodeStatusTimeout)
```

Comment:
Probably a good idea. This is one of the only two tests that require nodes to become ready again, and given that I didn't spend time understanding the implications for the other tests yet, I decided to keep the scope small.

Another possibility is to put this in the general AfterEach inside framework.go, possibly reducing flakes from other tests as well, but again it'd be harder to understand the scope of this.